opensearch/Dockerfile

```diff
@@ -24,4 +24,12 @@ LABEL git_branch=${GIT_BRANCH}
 LABEL git_commit=${GIT_COMMIT}
 LABEL git_diff=${GIT_DIFF}
 
-CMD ./sycamore-opensearch.sh
+USER root
+RUN yum install -yq openssl jq && yum clean all
+RUN chown 1000:1000 \
+    /usr/share/opensearch/sycamore-opensearch.sh \
+    /usr/share/opensearch/config/*.yml
+RUN chmod 600 /usr/share/opensearch/config/*.yml
```

Comment:
Does the chmod matter? I have learned that Docker is a bit stupid about stuff like this and near as I can tell this might just result in a layer which is a copy of the thing (that's what happens with chown).  Doesn't really matter given how large our images are, but seems better to not do if possible.

Reply:
OpenSearch complains when these files' modes are too permissive.

Reply:
`# Permissions must be restricted for opensearch to be happy`
It's not the number of layers it's the size of the layers.  You might think that when you just chown a file all that would be recorded would be a metadata change, but (to my surprise) Docker records the entire file. As a result our Jupyter image is about 2x the size it needs to be since the sycamore image has the files as root and the Jupyter one has them as 1000.  Given it does that for chown, I assume it does the same thing for chmod.  As I said, it doesn't really matter given the rest of our size so just adding the comment is sufficient, but it's worth other people understanding the Docker gotchas.